BCStudentSoftwareDevTeam/celts
# show rsvp history in track volunteers

## Combined with another issue, now in PR #879 
#829 

Did: 
- greyed out when a participant has unrsvped
- added a hover to show the rsvp history

Test:
- create an event
- change user to student (ayisie) and rsvp
- unrsvp
- go to events list, click on the event, click track volunteers, hover on the date and time under rsvp and you should see the rsvp history
